migration-tool/src/test/java/software/amazon/awssdk/migration/recipe/AddCommentToMethodTest.java

```diff
@@ -0,0 +1,90 @@
+import org.openrewrite.test.RewriteTest;
+import software.amazon.awssdk.migration.internal.recipe.AddCommentToMethod;
+
+public class AddCommentToMethodTest implements RewriteTest {
+
+    @Test
+    @EnabledOnJre({JRE.JAVA_8})
+    void shouldAddCommentToMethod() {
+        rewriteRun(
+            spec -> spec.recipe(new AddCommentToMethod("com.amazonaws.ClientConfiguration setCacheResponseMetadata(boolean)",
+                                                       "a comment"))
+                        .parser(Java8Parser.builder().classpath("sqs", "aws-core", "sdk-core", "aws-java-sdk-sqs")),
+            java(
+                "import com.amazonaws.ClientConfiguration;\n"
+                + "\n"
+                + "public class Example {\n"
+                + "    \n"
+                + "    void test() {\n"
+                + "        ClientConfiguration clientConfiguration = new ClientConfiguration();\n"
+                + "        clientConfiguration.setCacheResponseMetadata(false);\n"
+                + "    }\n"
+                + "}\n",
+                "import com.amazonaws.ClientConfiguration;\n"
+                + "\n"
+                + "public class Example {\n"
+                + "    \n"
+                + "    void test() {\n"
+                + "        ClientConfiguration clientConfiguration = new ClientConfiguration();\n"
+                + "        /*~~(a comment)~~>*/clientConfiguration.setCacheResponseMetadata(false);\n"
+                + "    }\n"
+                + "}"
+            )
+        );
+    }
+
+    @Test
+    @EnabledOnJre({JRE.JAVA_8})
+    void hasExistingComments_shouldNotClobber() {
+        rewriteRun(
+            spec -> spec.recipe(new AddCommentToMethod("com.amazonaws.ClientConfiguration setCacheResponseMetadata(boolean)",
+                                                       "a comment"))
+                        .parser(Java8Parser.builder().classpath("sqs", "aws-core", "sdk-core", "aws-java-sdk-sqs")),
+            java(
+                "import com.amazonaws.ClientConfiguration;\n"
+                + "\n"
+                + "public class Example {\n"
+                + "    \n"
+                + "    void test() {\n"
+                + "        ClientConfiguration clientConfiguration = new ClientConfiguration();\n"
+                + "        // Existing comment \n"
+                + "        /*existing comment*/ clientConfiguration.setCacheResponseMetadata(false);\n"
+                + "    }\n"
+                + "}\n",
+                "import com.amazonaws.ClientConfiguration;\n"
+                + "\n"
+                + "public class Example {\n"
+                + "    \n"
+                + "    void test() {\n"
+                + "        ClientConfiguration clientConfiguration = new ClientConfiguration();\n"
+                + "        // Existing comment \n"
+                + "        /*existing comment*/ /*~~(a comment)~~>*/clientConfiguration.setCacheResponseMetadata(false);\n"
```

Comment:
Does the `/* comment */` get rendered on its own line? Don't see a `\n` as above with the `// comment`

Reply:
It's rendered on the same line of the code. I followed the same style that openrewrite used for their recipes

Reply:
Got it, so this is just for test purposes. I assume we'd want the added comment on a separate line than the existing comment?

Reply:
Yeah, ideally, I tried to do that but it caused some other formatting issues; it's surprising hard to get the indent and formatting right after we add a new comment, so I just used one of openrewrite classes directly to generate the comment, which happens to be on the same line.